ee/app/engagement-dashboard/client/components/UsersTab/BusiestChatTimesSection.js

```diff
@@ -127,7 +127,7 @@ const ContentForDays = ({ displacement, onPreviousDateClick, onNextDateClick })
 	const { value: data } = useEndpointData('engagement-dashboard/users/chat-busier/weekly-data', params);
 	const values = useMemo(() => (data ? data.month.map(({ users, day, month, year }) => ({
 		users,
-		day: String(moment.utc([year, month - 1, day, 12, 0, 0]).valueOf()),
+		day: String(moment({ year, month: month - 1, day, hour: 0, minute: 0, second: 0 }).valueOf()),
```

Comment:
Since you're setting hour/minute/second to 0, you can skip those props.

Reply:
Great, thanks for the heads-up 🤗